app/src/main/cpp/skyline/services/am/controller/ICommonStateGetter.h

```diff
@@ -95,14 +95,21 @@ namespace skyline::service::am {
          */
         Result GetDefaultDisplayResolution(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
+        /**
+         * @brief Takes an input u32 #CpuBoostMode and passes it to SetCpuBoostMode.
```

Comment:
That description doesn't make much sense, `Sets the CPU boost mode to the supplied value` would make more sense.